go.mod

```diff
@@ -0,0 +1,5 @@
+module go.vocdoni.io/proto
```

Comment:
Why adding a go.mod file here? I'm not sure that's correct. What's the need for that?

Reply:
Needed the go.mod file locally to import from local dir this repo in the `vocdoni-node` repo. Can delete it from here if not needed.

Reply:
Isn't something like this enough on your go.mod?

`replace go.vocdoni.io/proto => ../dvote-protobuf`

Reply:
Got this error: `go: go.vocdoni.io/proto@v1.0.4-0.20210705131333-7925ca319268 (replaced by ../dvote-protobuf): reading ../../../dvote-protobuf/go.mod: open ../dvote-protobuf/go.mod: no such file or directory`

Reply:
I see, I can also reproduce it. @mvdan what would be the correct way to handle with it?

Reply:
Hmm yeah, I thought we already had a go.mod file. The replace from vocdoni-node won't work without one, because this isn't a module. Requiring `go.vocdoni.io/proto` from vocdoni-node worked because there's a fallback allowing to `go get` repos without a go.mod, for the transition period where many repos weren't modules yet.

One question is whether the module should be the entire repo, or just the `build/go` subdirectory. I guess the entire repo is easier, and it's not like it's particularly big either. Plus we're already using the entire repo as a module.